Userland/Libraries/LibWeb/Dump.cpp

```diff
@@ -772,6 +776,14 @@ ErrorOr<void> dump_supports_rule(StringBuilder& builder, CSS::CSSSupportsRule co
     return {};
 }
 
+ErrorOr<void> dump_property_rule( StringBuilder& builder, CSS::CSSPropertyRule const& property, int indent_levels)
+{
+    (void)builder;
+    (void)property;
+    (void)indent_levels;
+    return {};
```

Comment:
Please do implement this, it's helpful for debugging. Use `dump_font_face_rule()` as a guide.